### ISSUE TITLE
MIGRATION: Handle issues exceeding size limits

### ISSUE THREAD

**Issue by toddfarmer** [apache/arrow#14648]
While doing a dry run to test migration tooling, certain issues trigger errors during the import process due to excessive size of contents:

```
error: 413 Client Error: Payload Too Large for url: https://api.github.com/repos/toddfarmer/import_dry_run_2/import/issues
```

An example seems to be [this comment from ASF GitHub Bot](https://issues.apache.org/jira/browse/ARROW-1476?focusedCommentId=16254259&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16254259), which is a massive diff thrown into a comment.

We'll need to assess how to handle this. Some options include:

1. Arbitrarily truncate content exceeding thresholds during migration.
2. Skip importing content beyond threshold and leave a linked message back to the original content.
3. Identify and cleanse content before migration

**Comment by assignUser** [comment]
I'd drop the bot comments (the reason for their existence seems to no longer be the case, the diff is available via the PR) and fall back on truncating any other huge comments as a fallback.

**Comment by jorisvandenbossche** [comment]
Yes, regardless of the size issue, I also think we should not migrate those ASF GitHub Bot comments. 

For more recent comments of this bot, they are also no longer considered as "comments", but as activity. See for example https://issues.apache.org/jira/browse/ARROW-18173, you only see those comments if you look at "All" activity, and not if only looking at "Comments" activity. 
(so those type of comments are maybe already ignored?)
